config-default.yml

```diff
@@ -193,12 +194,10 @@ guild:
         helpers:            &HELPERS        385474242440986624
         incidents:                          714214212200562749
         incidents_archive:                  720668923636351037
-        mods:               &MODS           305126844661760000
+        mods:                               305126844661760000
         mod_alerts:                         473092532147060736
-        mod_appeals:        &MOD_APPEALS    808790025688711198
```

Comment:
Since those channels have been from here, they also need to be removed from `bot/constants.py`.